### PR TITLE
 Adding searchLib.sh, a script for sorting dependencies of librairies

### PR DESCRIPTION
**Pull request by fbailly**

### CODE REVIEW

File: scripts/searchLib.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    #echo $v
+    out_v=`basename $v`.txt
+    #echo $out_v
+    ldd $v  > /tmp/$out_v
+    grep -nH $1 /tmp/$out_v | grep -v $2 &> /tmp/test
+    if [ -s /tmp/test ]
+    then
+	echo $v
+    fi
+done
```
Comment (nim65s) [inline scripts/searchLib.sh]: I guess this could be written as:

```
ldd **.so | grep -nH $1 | grep -v $2
```

Or did I missed something ?

Comment (nim65s) [reply scripts/searchLib.sh]: what about
```
for v in **.so
do
  ldd $v | grep -nH $1 | grep -qv $2 && echo $v
done
```
?

Comment (fbailly) [reply scripts/searchLib.sh]: It works almost the same except it is not recursive. Maybe you'll find some more elegant way to do it  !

Comment (nim65s) [reply scripts/searchLib.sh]: `**` is recursive if you use `shopt -s globstar`. I think you can put that in your `.bashrc` and/or in this script

Comment (nim65s) [reply scripts/searchLib.sh]: My bad, in bash, you need to use `**/*.so` with `shopt -s globstar`

File: scripts/searchLib.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script searches all the .so files in the current directory.
+# The first arg is a string contained in the dependencies of the library you are looking for.
+# The second arg is a string contained in the dependencies that you want to reject. 
+# The script will display the names of the .so files which meet these criteria.
+# Ex: searchLib.sh my_lib version_to_avoid
```
Comment (nim65s) [inline scripts/searchLib.sh]: This is already copy/pasted in the README, so I think you should avoid to repeat yourself

**Pull request merged**